compiler-core/src/language_server/tests/compilation.rs

```diff
@@ -119,7 +119,7 @@ fn compile_recompile() {
     // This time it does not compile the module again, instead using the
     // cache from the previous run.
     let response = engine.compile_please();
-    assert!(response.result.is_ok());
+    assert_eq!(response.result, Ok(()));
```

Comment:
I got this test failing as a result of a typo, and the output wasn't too helpful. This makes it a lot better, so I thought I'd commit it.